LocationManagerExample/LocationManagerExample/INTUViewController.m

```diff
@@ -76,23 +92,12 @@ - (void)startLocationUpdateSubscription
         if (status == INTULocationStatusSuccess) {
             // A new updated location is available in currentLocation, and achievedAccuracy indicates how accurate this particular location is
             strongSelf.statusLabel.text = [NSString stringWithFormat:@"Subscription block called with Current Location:\n%@", currentLocation];
+            return;
         }
-        else {
```

Comment:
Please keep the `else` branch. It makes it very clear that a `Success` status is mutually exclusive with any other status. Having an early return doesn't make this as obvious.

The alternative would be to handle the error cases first, then early return -- this is a common pattern, however I prefer writing it the way it is written.